module/API/Controller.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file Controller.h
+ * @brief モーター制御に用いる関数をまとめたラッパークラス
+ * @author takahashitom
+ */
+#ifndef CONTROLLER_H
+#define CONTROLLER_H
+
+#include "ev3api.h"
+#include "Motor.h"
+
+class Controller {
+ public:
+  static const int MOTOR_PWM_MAX = 100;
+  static const int MOTOR_PWM_MIN = -100;
+
+  /**
+   * コンストラクタ
+   */
+  Controller();
+
+  /**
+   * タイヤのモータにPWM値をセット
```

Comment:
```suggestion
   * @brief タイヤのモータにPWM値をセット
```
各関数の説明は、こんな感じで書いてほしい
[参考記事](https://qiita.com/inabe49/items/23e615649e8539d857a8)